tests/README.md

```diff
@@ -74,10 +78,26 @@ make build-images load-images TAG=$(whoami)
 
 ## Step 3 - Run the tests
 
+### 3a - Run the tests locally
+
 ```makefile
 make test TAG=$(whoami)
 ```
 
+### 3b - Run the tests on a GKE cluster from a GCP VM
+
+This step only applies if you would like to run the tests from a GCP based VM. The VM should be created in the same
+zone as your GKE cluster, and requires a service account that has Kubernetes admin permissions. Additionally, you need
+ssh access to the VM and the VM needs to have network access to the Kubernetes control node.
+
+Before running the below `make` command, populate the required env vars in `utils/vars.env`.
+
+```makefile
+make run-tests-on-vm
```

Comment:
If I create my GKE cluster and run this script, is there anything else I need to do? Do I need to supply an ssh key, set up firewall policy, etc?

Reply:
I think it might depend on your configuration with GCP, but I don't need to provide any SSH key, it uses my cloud credentials I believe. The firewall policy is configured using tags - depending on what region you are deploying into, there may already be a rule you can use, otherwise you will need to create a new one. The rule name should be your TAG entry in the vars.env file (mine is `ciara-ssh` for example, but it's restrictive and only allows my IP in my region). You also need to provide a service account with the correct access - there is one available, I can provide the details offline.

Reply:
Can the script create a firewall policy that allows ssh access from our IP only?  We did something like this before for mesh where we used `$(curl -sS -4 icanhazip.com)/32` to get the IP address of the machine running the command. It would be nice if we didn't have to create anything manually.

Reply:
Done!